presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -134,26 +136,29 @@
     private final IDBI dbi;
     private final MetadataDao dao;
     private final ShardManager shardManager;
+    private final TypeManager typeManager;
```

Comment:
rather than putting `typeManager` on the stack, use `storageTypeConverter`

Reply:
From a high level point of view, this RaptorMetadata object is taking some other *manager objects so I think this is better. Also, you will be creating the storage type converter as needed so that's more efficient as we are putting the outer object only on the stack.